template_config.yml

```diff
@@ -60,6 +61,7 @@ pulp_settings:
   allowed_import_paths:
   - /tmp
 pulp_settings_azure: null
+pulp_settings_s3: null
 pulp_settings_s6: null
```

Comment:
This should be removed. It was a typo for s3.

Reply:
https://github.com/pulp/plugin_template/commit/c8a2852d99000f6018bb1172669d1ff0abec4a9a